components/slider/slider.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * MIGHTYminnow Components
+ *
+ * Component: Slider
+ *
+ * @package mm-components
+ * @since   1.0.0
+ */
+
+/**
+ * Build and return the Slider component.
+ *
+ * @since   1.0.0
+ *
+ * @param   array  $args  The args.
+ *
+ * @return  string        The HTML.
+ */
+function mm_slider( $args ) {
+
+	$component  = 'mm-slider';
+
+	// Set our defaults and use them as needed.
+	$defaults = array(
+		'image_ids'  => '',
+		'loop'       => '',
+		'autoplay'   => '',
+		'duration'   => '',
+		'navigation' => '',
+	);
+	$args = wp_parse_args( (array)$args, $defaults );
+
+	// Bail if we don't have any image IDs.
+	if ( empty( $args['image_ids'] ) ) {
+		return;
+	}
+
+	// Get clean param values.
+	$image_ids = ( is_array( $args['image_ids'] ) ) ? $args['image_ids'] : explode( ',', str_replace( ' ', '', $args['image_ids'] ) );
+
+	// Get Mm classes.
+	$mm_classes = apply_filters( 'mm_components_custom_classes', '', $component, $args );
+
+	ob_start() ?>
+
+	<div class="<?php echo esc_attr( $mm_classes ); ?>">
+		<ul>
+			<?php
+				foreach ( $image_ids as $image_id ) {
+
+					$image = wp_get_attachment_image_src( $image_id, 'full' );
+
+					if( is_wp_error( $image ) && is_array( $image ) ) {
```

Comment:
This check doesn't look right. I think we want ```if ( is_wp_error( $image ) || ! is_array( $image ) )```, which would be a check for whether wp_get_attachment_image_src worked (because it'll return an error if it fails and an array if it succeeds)